src/main/java/com/hotjava/app/hotjava/dao/PhotoDAO.java

```diff
@@ -10,30 +10,20 @@
 @Repository
 public class PhotoDAO implements IPhotoDAO {
 
-    //@Autowired
-   // private PhotoRepository photoRepository;
-
-   // @Autowired
-    //private KafkaTemplate<String, String> kafkaTemplate;
-
-
-
-
     @Override
     public void save(Photo photo) {
     //TODO photo repo
     }
 
     @Override
     public void saveImage(MultipartFile imageFile, Photo photo) throws IOException {
-       /* Path currentPath = Paths.get(".");
-        Path absolutePath = currentPath.toAbsolutePath();
-        photo.setPath(absolutePath + "/src/main/resources/static/photos/");
-        byte[] bytes = imageFile.getBytes();
-        Path path = Paths.get(photo.getPath() + imageFile.getOriginalFilename());
-        Files.write(path, bytes);
-        kafkaTemplate.send("photoIn", path.normalize().toString());*/
-        System.out.println("saveImage" + Integer.toString(photo.getPhotoId()));
+        try {
+            System.out.println("saveImage " + photo.getPhotoId());
+        }
+        catch(Exception e) {
+            e.printStackTrace();
```

Comment:
In this case, you're catching the exception, then ignoring it, and then acting as if everything worked fine on line 26.  That hides very important information about what went wrong.

Suggestions:
- Try to heal the error, instead of throwing it away and ignoring it.
- Use proper logging, with logging levels.  No one will see e.printStackTrace()